api/tests/unit/tooling/chai-custom-helpers/deep-equal-array_test.js

```diff
@@ -6,6 +6,7 @@ describe('Unit | chai-custom-helpers | deepEqualArray', function () {
       expect([]).to.deepEqualArray('coucou');
     }, "expected 'String' to equal 'Array'");
     global.chaiErr(function () {
+      // eslint-disable-next-line chai-expect/no-inner-literal
       expect('coucou').to.deepEqualArray([]);
```

Comment:
C'est un peu dommage de le désactiver juste pour ça : 
```suggestion
      const foo = 'bar'
      expect(foo).to.deepEqualArray([]);
```

Reply:
Bien vu, corrigé